packages/go_router_builder/test/test_inputs/_go_router_builder_test_input.dart

```diff
@@ -325,3 +325,28 @@ class IterableDefaultValueRoute extends GoRouteData {
   IterableDefaultValueRoute({this.param = const <int>[0]});
   final Iterable<int> param;
 }
+
+@ShouldGenerate(r'''
+RouteBase get $namedRoute => GoRouteData.$route(
+      path: '/named-route',
+      name: 'namedRoute',
```

Comment:
Can you also add a test for with name that requires dart string escape?

Reply:
Good point, sure!